dockertest/docker_daemon.py

```diff
@@ -126,11 +126,17 @@ def which_docker():
     possible, but unsupported, so we're not going to worry about it.
     """
     docker = 'docker'
-    with open('/etc/sysconfig/docker', 'r') as docker_sysconfig:
-        for line in docker_sysconfig:
-            if line.startswith('DOCKERBINARY='):
-                if 'docker-latest' in line:
-                    docker = 'docker-latest'
+
+    # /etc/sysconfig/docker doesn't exist on the CI host on which
+    # we run github unit tests
+    try:
+        with open('/etc/sysconfig/docker', 'r') as docker_sysconfig:
```

Comment:
One idea could be to mock the open call (which I'm told is hard).  Another idea is to indirect reference the open-call, to support unit-tests modifying it:

```python
OPEN_FN = open
...

    def which_docker():
        ...
        with OPEN_FN(...):
            ...
```